cmake/toolchain/zephyr/0.10.0/target.cmake

```diff
@@ -28,3 +27,4 @@ if("${ARCH}" STREQUAL "xtensa")
   LIST(APPEND TOOLCHAIN_LIBS hal)
   LIST(APPEND LIB_INCLUDE_DIR -L${SYSROOT_DIR}/lib)
 endif()
+set(SYSROOT_DIR   ${TOOLCHAIN_HOME}/${SYSROOT_TARGET}/${SYSROOT_TARGET})
```

Comment:
wait, this change applies to all architectures, why the title talks about xtensa?

Reply:
Because xtensa is the only one impacted.  All the others had this happening before.